### PR TITLE
AEROGEAR-2244 Mobile app security dashboard

### PR DESCRIPTION
**Pull request by aliok**
After the discussion in https://groups.google.com/forum/#!topic/aerogear/MgFcW2gF_oI , this is the new dashboard.

Prerequisites:
- `event_type` column in `mobileAppMetrics` table
- Security events should contain app and device data as well (everything in init event's data basically)

Work done:
- Use of event types in existing "mobile services" and "mobile app metrics" dashboards
- Addition of mobile security overview row in mobile services dashboard
- New dashboard: mobile security

Screenshots:

Mobile services modified:
screenshot from 2018-04-03 15-13-51

Mobile security:
screenshot from 2018-04-03 15-14-12

screenshot from 2018-04-03 15-14-22

### CODE REVIEW

**Comment by aliok** [comment]
cc @paolobueno can't add you as reviewer

**Comment by david-martin** [comment]
@aliok would you have an updated script for pushing random data to the server?
Also, is there a server change needed to store the event_type field?
(I'm guessing that is a new field to be send by the SDKs?)

**Comment by aliok** [comment]
@david-martin yeah it is a bit hard to verify right now. please just have a look at the screenshots for now. @paolobueno can do the verification later when the necessary changes are done in other components.

File: roles/provision-metrics-apb/files/mobile-app-security-dashboard.json [hunk 1]
```diff
@@ -0,0 +1,970 @@
...
+      "keepTime": true,
+      "tags": [
+        "mobile-services"
+      ],
+      "type": "dashboards"
+    }
+  ],
+  "panels": [
+    {
+      "collapsed": false,
```
Comment (aliok) [inline roles/provision-metrics-apb/files/mobile-app-security-dashboard.json]: I will fix this

**Comment by aliok** [comment]
Merging this one even https://github.com/aerogear/aerogear-app-metrics/pull/52 is not merged yet. I don't expect any problems.

**Pull request merged**